meta/runtime.yml

```diff
@@ -1,2 +1,9 @@
 ---
 requires_ansible: '>=2.15.0'
+plugin_routing:
+    modules:
+        vm_list_group_by_clusters:
+            redirect: vmware.vmware.vm_list_group_by_clusters_info
+            deprecation:
+                removal_version: 2.0.0
+                warning_text: Use vmware.vmware.vm_list_group_by_clusters_info instead.
```

Comment:
I haven't renamed a module for quite some time. But since you're `plugin_routing` here, do we _also_ need `plugins/modules/_vm_list_group_by_clusters.py`? I should say no, but I honestly don't know :shrug:

Reply:
As far as I understood from that [page](https://docs.ansible.com/ansible/latest/dev_guide/module_lifecycle.html#changing-a-module-or-plugin-name-in-the-ansible-main-repository) we need but I'm not sure
 @Andersson007 what do you think?

Reply:
I don't know two, i would look in other collections how it's done there. Maybe would switch between branches in community.general and see

Reply:
So it seems that it's not recommended anymore to add the "_" prefix, Also I got some errors regarding that when I used to add the "_vm_list_group_by_clusters" instead of "vm_list_group_by_clusters"

Reply:
Underscores in collections do *not* deprecate. That was never really the case.

Reply:
I'm agree with you but I followed after the [documentation](https://docs.ansible.com/ansible/latest/dev_guide/module_lifecycle.html#the-lifecycle-of-an-ansible-module-or-plugin ) : 

> Rename the file so it starts with an _, for example, rename old_cloud.py to _old_cloud.py. This keeps the module available and marks it as deprecated on the module index pages. 

Also, this is a[ PR](https://github.com/ansible/ansible/pull/43781/files) example that was added to the documentation and it seems to include the underscore.

Reply:
I'm resolving those comments since we agree not to add underscore.

Reply:
The beginning of the section you are quoting (https://docs.ansible.com/ansible/latest/dev_guide/module_lifecycle.html#deprecating-modules-and-plugins-in-the-ansible-main-repository) says "To deprecate a module **in ansible-core,** you must:" (emphasis mine). The section after that is for deprecating a module or plugin in a collection, and it doesn't mention underscores.